DataConnectors/SentinelOne/SentinelOneSentinelConnector/__init__.py

```diff
@@ -120,6 +120,7 @@ def get_report(self, report_type_suffix, report_type_name, params = None):
                 logging.error("Something wrong. Error code: {}".format(r.status_code))
         except Exception as err:
             logging.error("Something wrong. Exception error text: {}".format(err))
+            logging.debug("Status code for reports: " + str(r.status_code))

```

Comment:
Use:
logging.debug("Status code for reports:  %s" % r.status_code)

Reason:
https://dev.to/izabelakowal/what-is-the-best-string-formatting-technique-for-logging-in-python-d1d